azurerm/internal/services/compute/linux_virtual_machine_resource.go

```diff
@@ -401,6 +412,11 @@ func resourceLinuxVirtualMachineCreate(d *schema.ResourceData, meta interface{})
 		Tags: tags.Expand(t),
 	}
 
+	if v, ok := d.GetOk("license_type"); ok {
+		license := v.(string)
+		params.VirtualMachineProperties.LicenseType = &license
```

Comment:
Thoughts on shortening this to
```suggestion
		params.VirtualMachineProperties.LicenseType = utils.String(v.(string))
```